added meta for guides (#39)
dashpay/docs-core: docs/guide/dash-features-masternode-quorums.md

```{eval-rst}
.. meta::
  :title: Masternode Quorums

>   :description: Dash’s masternode quorums are used to facilitate the operation of masternode provided features in a decentralized, deterministic way. 

Reviewer: ```suggestion
  :description: Dash’s masternode quorums enable masternode-provided features to work in a decentralized, deterministic way. 
```
